main.py

```diff
@@ -40,13 +40,13 @@ async def send_pages(self):
 client.help_command = MyHelpCommand()
 
 
-@client.command(name="getUTC")
+@client.command(name="getUTC", help="Get the current UTC time.")
 async def getUTC(ctx) -> None:
     utc_time = await getUTCTime()
     await ctx.send(utc_time.strftime("%Y-%m-%d %H:%M:%S"))
 
 
-@client.command(name="fetch")
+@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: !fetch <param>\nAllowed params: sig, life, tor, wind, hail")
```

Comment:
Current code uses $ for the prefix.
$fetch sig tor
$fetch tor are both proper 

$fetch life hasn't been implemented (yet)

Reply:
Ah my brain stopped working, yeah it should be $ instead